frontend/packages/console-shared/src/components/catalog/catalog-view/CatalogFilters.tsx

```diff
@@ -24,6 +24,13 @@ const CatalogFilters: React.FC<CatalogFiltersProps> = ({
   onFilterChange,
   onShowAllToggle,
 }) => {
+  const sortedActiveFilters = Object.keys(activeFilters)
+    .sort()
+    .reduce((acc, groupName) => {
```

Comment:
"I like types" :smirk: without defining it at the reduce or the initial parameter of the reduce its just `{}`

```suggestion
    .reduce<CatalogFilters>((acc, groupName) => {
```